assets/js/components/RoundedPicture/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {fetchUserData} from '../../user/redux'
+
+class RoundedPicture extends Component {
+    componentDidMount() {
+        this.props.fetchUserData()
+    }
+
+    render() {
+        const {user} = this.props
+
+        return (
+            <div className="greetings-user-image" >
```

Comment:
This components name is RoundedPicture so I think it would be clearer if this class would reflect that, e.g. "rounded-image"